sabnzbd/newswrapper.py

```diff
@@ -279,7 +280,18 @@ def __init__(self, nw: NewsWrapper, host):
         # Create SSL-context if it is needed and not created yet
         if self.nw.server.ssl and not self.nw.server.ssl_context:
             # Setup the SSL socket
-            self.nw.server.ssl_context = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
+            try:
+                self.nw.server.ssl_context = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
+            except ssl.SSLError:
+                logging.debug("Broken SSL, trying workaround")
+                self.nw.server.ssl_context = ssl.SSLContext()
```

Comment:
Python [docs](https://docs.python.org/3/library/ssl.html#ssl.SSLContext) say that _since version 3.10 [...] SSLContext without protocol argument is deprecated. The context class will either require PROTOCOL_TLS_CLIENT or PROTOCOL_TLS_SERVER protocol in the future._